website/docs/patterns/form-patterns/partials/validation/validation.md

```diff
@@ -47,21 +44,51 @@ For this reason, we recommend creating your validation mechanism using our form
 
 !!!
 
-### Validation on focus change
+### Form-level validation
+
+Form-level validation should be used for server-side errors that cannot be checked before data submission or requires additional checks after data submission. This error type should appear at the top of the form and use a Critical Inline [Alert](/components/alert). Using an Alert makes the message a focal point for users to see what occurred during the submission to result in an error. 
 
-Validation on focus change occurs on the client and refers to validating a field when it loses focus via an [`onblur`](https://developer.mozilla.org/en-US/docs/Web/API/Element/blur_event) event.
+For form-level validation that identifies multiple input-level errors, the alert should contain a message listing **all** errors with links to each invalid field. We recommend using an introductory sentence to give context before listing the errored items, e.g., “Please correct the following errors and resubmit the form”
 
-This method is immediate in the sense that it displays an error directly after a field has been filled out, while not interrupting the user while they are in the process of filling out a field.
+!!! Do
+
+Use form-level validation for overarching form submission errors or for a list of server-side validation errors that link to their respective errored fields.
+
+![Sign in page with authentication error alert at the form level](/assets/patterns/form-patterns/do-form-level-error.png)
+
+!!!
+
+!!! Dont
+
+Never use form-level alerts for  field-specific, inline validations. 
+
+![Credit card info form with a single field-level error and duplicative form-level alert](/assets/patterns/form-patterns/dont-form-level-error.png) 
+
+!!!
 
-Use this method **whenever possible** to resolve an error while the user is still in the context of the erroring field. This can help limit the amount of context switching necessary when filling out a form and reduce the cognitive load on the user.
+
+### Combining inline and form-level validation
+
+We recommend always using inline validation to provide immediate feedback, but for complex forms that require both client-side and server-side checks, a combined approach is best. 
+
+![Form with multiple field-level errors and a form-level alert with jump links to each](assets/patterns/form-patterns/dont-form-level-error.png)
+
+
+## Timing validation messages
+
+### Validation on focus change
+
+Client-side validation occurs when a user leaves the field via an [onblur](https://developer.mozilla.org/en-US/docs/Web/API/Element/blur_event) event. This immediate feedback communicates errors as they happen.
+
+Because this method minimizes interruption, use it **whenever possible** so users can resolve errors while still in the context of the form. This can limit the amount of context switching necessary to complete the form successfully and reduce the cognitive load on the user.
```

Comment:
```suggestion
Because this method minimizes interruption, use it whenever possible so users can resolve errors while still in the context of the form. This can limit the amount of context switching necessary to complete the form successfully and reduce the cognitive load on the user.
```